crates/symbolicator/src/services/download/mod.rs

```diff
@@ -238,6 +250,11 @@ impl DownloadService {
 /// Download the source from a stream.
 ///
 /// This is common functionality used by many downloaders.
+///
+/// # Errors
+/// - [`DownloadError::BadDestination`]
+/// - [`DownloadError::Write`]
+/// - [`DownloadError::Cancelled`]
```

Comment:
```suggestion
/// - [`DownloadError::Canceled`]
```

Reply:
I'm now really curious how one spells cancelled in Canada

Reply:
With two l's! `Canceled` precedes me so it looks like the person who added it used the US spelling, but there are logs that use double l's

Reply:
Ooh, interesting.  British is double-L as well, but american single-L.  I've been trying to stick to american for code as that was already case, though for comments (and probably logs) I stay true to my beliefs 😄  That probably explains why some double-L use already exists.